contracts/kernel/Kernel.sol

```diff
@@ -30,60 +34,59 @@ contract Kernel is KernelStorage, Initializable {
     */
     function initialize(address _permissionsCreator) onlyInit {
         initialized();
-        bytes4 createPermissionAction = bytes4(sha3("createPermission(address,address,bytes4,address)"));
-        _createPermission(_permissionsCreator, address(this), createPermissionAction, _permissionsCreator);
+        _createPermission(_permissionsCreator, address(this), PERMISSION_CREATOR_ROLE, _permissionsCreator);
     }
 
     /**
     * @dev Creates a permission that wasn't previously set. Access is limited by the ACL.
     *      if a created permission is removed it is possible to reset it with createPermission.
-    * @notice Create a new permission granting `_entity` the ability to perform `_action` on `_app` (setting `_parent` as parent)
-    * @param _entity Address of the whitelisted entity that will be able to perform the action
-    * @param _app Address of the app in which the action will be allowed (requires app to depend on kernel for ACL)
-    * @param _action Function signature of the action (requires action to be protected with the `auth` modifier)
+    * @notice Create a new permission granting `_entity` the ability to perform `_role` on `_app` (setting `_parent` as parent)
+    * @param _entity Address of the whitelisted entity that will be able to perform the role
+    * @param _app Address of the app in which the role will be allowed (requires app to depend on kernel for ACL)
+    * @param _role Function signature of the role (requires role to be protected with the `auth` modifier)
```

Comment:
Not actually a role signature, just the role identifier bytes, also *requires role to be protected with the `auth` modifier* makes no sense.